required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -2516,6 +2516,13 @@
         \advance \linewidth -\leftmargin
         \parshape \@ne \leftmargin \linewidth
         \footnotesize
+        \parskip\hangfootparskip\relax
+%    \end{macrocode}
+%    Setting of \cs{parindent} has to happen prior to calling
+%    \cs{leavevmode} (which happens below to support
```

Comment:
Maybe better:
The setting of . . . 

And:
How can anything "happen below"? 
We are not climbing a mountain, are we? 

Similarly. plesse:
moved up.  ==>  moved to earlier.

Reply:
There is some bias in it (reading from top to bottom) but this is what we do.